metaserver/src/main/scala/scala/meta/languageserver/compiler/SignatureHelpProvider.scala

```diff
@@ -16,7 +16,7 @@ object SignatureHelpProvider extends LazyLogging {
       global = compiler,
       code = cursor.contents,
       filename = cursor.uri,
-      cursor = None
+      cursor = Some(cursor.offset + 1)
```

Comment:
`offset + 1` because we want

```
Predef.assert(_CURSOR_
```

and not

```
Predef.assert_CURSOR_(
```

Reply:
Did you check if this changed/improved the behavior anywhere?

Reply:
Honestly, no. I just made sure all tests still pass (and the first one didn't without the `+ 1`

Reply:
Sounds good, I will keep my eyes open for what effect this has, maybe it's not as important for signatureHelp as I thought it would be